source/getting-started-edge/index.rst

```diff
@@ -0,0 +1,245 @@
+
+.. _InCommon: https://incommon.org/federation
+
+.. _getting-started:
+
+=============================
+Getting started with CHI@EDGE
+=============================
+
+This guide will walk you through the initial steps of getting an account,
+joining a project and working with your first container.
+
+.. contents:: :local:
```

Comment:
What does this do?

Reply:
This generates a TOC for the local page, based on the headers, which is also on the normal getting started page. Here is a screenshot:
![image](https://user-images.githubusercontent.com/9397092/120835359-d6064c00-c529-11eb-8930-a32ec258d14e.png)